packages/driver/src/cy/actionability.ts

```diff
@@ -64,8 +64,12 @@ const ensureElIsNotCovered = function (cy, win, $el, fromElViewport, options, lo
     elAtCoords = $dom.getElementAtPointFromViewport(win.document, fromElViewport.x, fromElViewport.y)
 
     if (elAtCoords) {
-      return $elAtCoords = $dom.wrap(elAtCoords)
+      $elAtCoords = $dom.wrap(elAtCoords)
+
+      return $elAtCoords
     }
+
+    return null
```

Comment:
It should be null because `$elAtCoords` starts with `null` at line 57. If we change it to `undefined`, `$elAtCoords` assignment will fail.

To make it work, we should change the type to `JQuery<any> | null | undefined`. But `null | undefined` is weird because there's no difference between `null` and `undefined` in this case.